amazon/sqs/client/src/main/kotlin/org/http4k/connect/amazon/sqs/HttpSQS.kt

```diff
@@ -24,7 +24,7 @@ fun SQS.Companion.Http(
 ) = object : SQS {
     private val signedHttp = signAwsRequests(region, credentialsProvider, clock, Signed, overrideEndpoint).then(http)
 
-    override fun <R> invoke(action: SQSAction<R>) = action.toResult(signedHttp(action.toRequest()))
```

Comment:
Not a huge fan of this, but adding the second type argument lets us maintain backwards compatibility for the output type, which may differ from the response type to be deserialized.